Software/Development/SemesterProject/App/Include/Constants.h

```diff
@@ -8,77 +8,77 @@
 
 
 //Processor related parameters
-#define CLOCK_FREQUENCY ((double)150e6)     // 150 MHz clock frequency
-#define CLOCK_PERIOD    ((double)6.667e-9 ) // 1/CLOCK_FREQUENCY clock period
-#define SW_FREQ      10000          //Switching frequency [10kHz].
-#define SW_PERIOD_US   100           //Period of the PWM.
-#define SW_PERIOD_S    ((float)100e-6)   //PWM period in s.
-#define INTERNAL_FREQ   90000000    //Internal frequency of the device [90MHz]. Configured in InitSysCtrl().
+#define CLOCK_FREQUENCY             ((float)150e6)          // 150 MHz clock frequency
+#define CLOCK_PERIOD                ((float)6.667e-9)       // 1/CLOCK_FREQUENCY clock period
+#define SW_FREQ                             10000           //Switching frequency [10kHz].
+#define SW_PERIOD_US                        100             //Period of the PWM.
+#define SW_PERIOD_S                 ((float)100e-6)         //PWM period in s.
+#define INTERNAL_FREQ                       90000000        //Internal frequency of the device [90MHz]. Configured in InitSysCtrl().
 
-#define MinPulseWidth	((double)2e-6) //minimum pulse width in s
-#define MinPulseWidthCNT	((unsigned short)300)        //MinPulseWidth/CLOCK_PERIOD (2us)
-#define Half_MinPulseWidthCNT	((unsigned short)150)	//half MinPulseWidth/CLOCK_PERIOD (1us)	
+#define MinPulseWidth	            ((float)2e-6)           //minimum pulse width in s
+#define MinPulseWidthCNT	        ((unsigned short)300)   //MinPulseWidth/CLOCK_PERIOD (2us)
+#define Half_MinPulseWidthCNT	    ((unsigned short)150)	//half MinPulseWidth/CLOCK_PERIOD (1us)
```

Comment:
WHy are these not aligned?

Reply:
I really tried to do so, but looks like that didn't go so well... I will try again